src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain
+
+class Player(
+    val name: String,
+    val hand: Hand = Hand()
+) {
+
+    init {
+        require(name.isNotBlank()) { "플레이어의 이름은 공백일 수 없습니다." }
+    }
+
+    fun init(deck: Deck) {
+        val initCards = deck.init()
+        hand.init(initCards[0], initCards[1])
+    }
```

Comment:
```suggestion
    fun init(cards:List<Card>) {
        require(card == 2) {}
        hand.init(cards[0], cards[1])
    }
```

카드를 두개 받고, 이에 대해서 검증해주는 로직으로 가는건 어떨까요?

Reply:
넵 수정하겠습니다!